dist/lib/connection.js

```diff
@@ -203,7 +203,9 @@ class Connection extends _eventemitter.default {
   }
 
   initiateSocket() {
-    this.socket = new _isomorphicWs.default(this.wsURI);
+    this.socket = new _isomorphicWs.default(this.wsURI, this.options.rejectUnauthorized === false ? {
```

Comment:
How is this related? No changes in src/lib/connection.js

Reply:
@tkurki I reckon that the last time I pushed to master (PR or otherwise) no dist was made. Having said that; any changes to the `dist` folder can be ignored -- as long as `src` is good dist will follow. I'll look at the build scripts to make that more automated

Reply:
Probably a commit hook to build before commit would be useful here